cli/utilities/packages.py

```diff
@@ -201,6 +207,27 @@ def register(self, username, password, serverurl=None, baseurl=None, force=False
                 "Failed to register node(s) to subscription manager"
             )
 
+    def clean(self):
+        """Cleans all local subscription data: entitlements, certs, identity.
+
+        Raises:
+            SubscriptionManagerError: if clean fails.
+        """
+        try:
+            self.unregister()
```

Comment:
same as above. handle the unregister wherever required. let us not put it in utils module.

Reply:
done